docs/README.md

```diff
@@ -37,8 +37,8 @@ Make use of the many generators for code, try `ember help generate` for more det
 
 ### Linting
 
-* `yarn lint`
-* `yarn lint:fix`
+* `pnpm lint`
+* `pnpm lint:fix`
```

Comment:
To execute a script, you need to use `pnpm run` as far as I know.


```suggestion
* `pnpm run lint`
* `pnpm run lint:fix`
```

Reply:
according to their site (https://pnpm.io/pnpm-cli#commands)

`npm run <cmd>` is equivalent to `pnpm <cmd>`

I do think using `pnpm run` will work, so I can change it if you would prefer that, but I did try it locally and `pnpm lint` works as expected.

Reply:
See above.